City-of-Helsinki/drupal-helfi-etusivu
# UHF-5665: Current right now -paragraph

# [UHF-5664: Current right now -paragraph](https://helsinkisolutionoffice.atlassian.net/browse/UHF-5665)

## What was done?
Added new paragraph type `current`
Added styles for the paragraph

### How to install?
- Checkout branch. Run `make fresh`
- Run `drush cr; drush locale-check; drush locale-update`

### How to test?
- Add or edit a landing page. You should be able to add a `current` paragraph to content area.
- Fill the fields and save the node. View the page.

Check that the paragraph matches the sketch: https://share.goabstract.com/ca21840a-654f-4386-a828-93dd3759040d?collectionLayerId=562a3bb6-2640-4fa5-95dc-c3e37cc9e2f4&mode=design

**Note!** In mobile, the seasonal illustration is below the text. This does not match the sketch currently, but it has been discussed with and approved by the designer.
